src/lib/index.ts

```diff
@@ -144,6 +144,30 @@ class ServerlessIamPerFunctionPlugin {
     return length;
   }
 
+  /**
+   * @param {*} functionObject
+   * @returns {boolean}
+   */
+  hasPermissionsBoundaryDefined(functionObject: any): boolean {
+    return !_.isEmpty(functionObject.iamPermissionsBoundary) || this.hasGlobalPermissionsBoundaryDefined();
+  }
+
+  /**
+   * @returns {boolean}
+   */
+  hasGlobalPermissionsBoundaryDefined(): boolean {
+    const customIamRolesPerFunction = this.getCustomIamRolesPerFunction();
+
+    return !_.isEmpty(customIamRolesPerFunction) && !_.isEmpty(customIamRolesPerFunction.iamGlobalPermissionsBoundary);
+  }
+
+  /**
+   * @returns {*}
+   */
+  getCustomIamRolesPerFunction(): any {
+    return this.serverless.service.custom['serverless-iam-roles-per-function'];
```

Comment:
1. `custom` node might not be defined at all. As a result you will have error:
`VM440:1 Uncaught TypeError: Cannot read property 'serverless-iam-roles-per-function' of undefined`
Consider to replace with ```_.get(this.serverless.service, `custom.${PLUGIN_NAME}`, false);```
You can find example [here](https://github.com/functionalone/serverless-iam-roles-per-function/blob/v3.0.2/src/lib/index.ts#L68)

Reply:
can I do only this?
```js
_.get(this.serverless.service, `custom.${PLUGIN_NAME}`);
```

I prefer to return undefined over false when nothing is defined, do you see any  advantage in returning false?

Reply:
`undefined` works for me, but in my opinion, all the helper methods (`getCustomIamRolesPerFunction ` and `hasGlobalPermissionsBoundaryDefined `) might be replaced with a single ```_.get(this.serverless.service, `custom.${PLUGIN_NAME}. iamGlobalPermissionsBoundary`);```

Reply:
Like:
```
const iamGlobalPermissionsBoundary = _.get(this.serverless.service, `custom.${PLUGIN_NAME}.iamGlobalPermissionsBoundary`);

if (iamGlobalPermissionsBoundary) {
 const assignToMe = iamGlobalPermissionsBoundary
}
```

Reply:
You are right, I removed all helper methods.